packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -438,6 +438,9 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
 
         // Prevent depositing transactions that have too small of a gas limit.
         require(_gasLimit >= 21_000, "OptimismPortal: gas limit must cover instrinsic gas cost");
+        // Prevent the creation of deposit transactions that have too much calldata. This gives an
+        // upper limit on the size of unsafe blocks over the p2p network.
+        require(_data.length <= 120_000, "OptimismPortal: data too large");
```

Comment:
I added a comment with the reasoning behind the number that was chosen. Its definitely a magic number, the idea is in case we want to reuse p2p networking stack for deposit transactions eventually